project/CommonSettings.scala

```diff
@@ -139,6 +142,20 @@ object CommonSettings {
 
   lazy val prodSettings: Seq[Setting[_]] = settings
 
+  lazy val appSettings: Seq[Setting[_]] = prodSettings ++ Vector(
+    //gives us the 'universal' directory in build artifacts
+    Compile / unmanagedResourceDirectories += baseDirectory.value / "src" / "universal"
+  )
+
+  lazy val dockerSettings: Seq[Setting[_]] = {
+    Vector(
+      //https://sbt-native-packager.readthedocs.io/en/latest/formats/docker.html
+      dockerBaseImage := "openjdk",
```

Comment:
Are we able to / should we define a java version >11 so if in the future we are using something with eclair it will work?

Reply:
It seems the `openjdk` policy is to use the latest release of java, so this is using `openjdk-15` as of now (see layer 4)

https://hub.docker.com/layers/138104531/bitcoinscala/bitcoin-s/latest/images/sha256-35e35532d821abc257399136252a8ff20a679fec2545e31c80b5a0f95f02c8e0?context=explore